hugo/content/exporter/_index.md

```diff
@@ -463,7 +463,9 @@ The following metrics either require special considerations when monitoring spec
  
  * *ccp_connection_stats_max_blocked_query_time* - Runtime of the longest running query that has been blocked by a heavyweight lock
 
- * *ccp_replication_lag_replay_time* - Only provides values on replica instances. Time since replica received and replayed a WAL file. Note this is not the main way to determine if a replica is behind its primary. It only monitors the time the replica replayed the WAL vs what it has received. It is a secondary metric for monitoring WAL replay on the replica itself.
+ * *ccp_replication_lag_replay_time* - Only provides values on replica instances. Time since replica received and replayed a WAL file. Note this is not the main way to determine if a replica is behind its primary. It only monitors the time since the replica replayed the WAL vs what it has received. It also does not monitor when a WAL replay replica completely stops receiving WAL (see received_time metric). It is a secondary metric for monitoring WAL replay on the replica itself.
+
+ * *ccp_replication_lag_received_time* - Similar to *ccp_replication_lag_replay_time*, however this value always increases until the replica actually receives and replays a WAL file. Effective for monitoring that a WAL replay replica has actually received WAL files. Note this will cause false positives when used as an alert for replica lag if the primary receives no writes.
```

Comment:
```suggestion
 * *ccp_replication_lag_received_time* - Similar to *ccp_replication_lag_replay_time*, however this value always increases between receipt of WAL files. Effective for monitoring that a WAL replay replica has actually received WAL files. Note this will cause false positives when used as an alert for replica lag if the primary receives no writes (which means there is no WAL to send).
```

Reply:
So while the name of this metric is "received" it is still technically the receipt AND replay of the WAL file. There is no metric to track just the receipt from the replica's side at the moment. I just had to think of a different name for it vs the other metric.